example/README.md

```diff
@@ -0,0 +1,82 @@
+# FluentD to Dynatrace Log Ingest
+
+It is possible to have FluentD ingest the logs of your kubernetes nodes and let FluentD send the resulting logs to your Dynatrace environment. See below for instructions.
+
+## Prerequisites
+
+Generally needed:
+
+- A Dynatrace namespace, if you do not already have one
+- An [API Token](https://www.dynatrace.com/support/help/dynatrace-api/basics/dynatrace-api-authentication/) with "Ingest logs" permission
+- The dynatrace-monitoring service-account
+
+   ``kubectl apply -f service-account.yaml``
+
+Only needed in combination with a Dynatrace ActiveGate:
+
+- A [PaaS Token](https://www.dynatrace.com/support/help/get-started/access-tokens/)
+- Available persistent storage in your Kubernetes cluster
+
+## Decide on a log ingest endpoint
+
+You have two options when choosing the endpoint fluentd sends your log to.
+This is either an ActiveGate, which might already exist in your environment or a pre-existing log ingest on your SaaS Dynatrace environment.
+
+### Option 1: Deploy an ActiveGate
+
+1. Create a secret holding the environment URL and login credentials for this registry, making sure to replace
+
+   - ${YOUR_ENVIRONMENT_URL} with your environment URL (without 'http'). Example: environment.live.dynatrace.com
+   - ${YOUR_ENVIRONMENT_ID} with the ID in your environment URL
+   - ${YOUR_PAAS_TOKEN} with the PaaS token you created in Prerequisites
```

Comment:
You could put all of these placeholders in backticks as well (`${YOUR_ENVIRONMENT_URL}`, ...) for consistent formatting (same below).